workshops/views.py

```diff
@@ -1061,16 +1062,19 @@ def event_import(request):
 
         if 'slug' not in tags:
             # there are no HTML tags, so let's try the old method
-            url = generate_url_to_event_index(url)
+            index_url = generate_url_to_event_index(url)
 
             # fetch page
-            response = requests.get(url)
+            response = requests.get(index_url)
 
             if response.status_code == 200:
                 # don't throw errors for pages we fall back to
                 content = response.text
                 tags = find_tags_on_event_index(content)
 
+                if 'slug' not in tags:
+                    tags['slug'] = urlparse(url).path.replace('/', '')
+
         # normalize (parse) them
         tags = parse_tags_from_event_website(tags)
 
```

Comment:
On Mon, Nov 23, 2015 at 01:39:48PM -0800, Piotr Banaszkiewicz wrote:
> `parse_tags_from_event_website` is insensitive to empty
> dictionaries, see for example `test_parsing_empty_tags` in
> `test_util.py`.

“We didn't find any tags, because that URL 404ed” is going to be more
useful to admins than “We didn't get any tags, good luck!” ;).